src/main.js

```diff
@@ -105,22 +97,22 @@ function homeVisibility() {
   saveCoverButton.classList.remove("hidden");
 };
 
+function pushAndSetFieldValues() {
+  covers.push(coverField.value);
+  titles.push(titleField.value);
+  descriptors.push(desc1Field.value, desc2Field.value);
+  coverImg.setAttribute("src", coverField.value);
+  title.innerText = titleField.value;
+  descriptor1.innerText = desc1Field.value;
+  descriptor2.innerText = desc2Field.value;
+};
+
 function createNewBook(event) {
   event.preventDefault();
-  var coverValue = coverField.value;
-  var titleValue = titleField.value;
-  var desc1Value = desc1Field.value;
-  var desc2Value = desc2Field.value;
-  if (coverValue && titleValue && desc1Value && desc2Value) {
+  if (coverField.value && titleField.value && desc1Field.value && desc2Field.value) {
     homeVisibility();
```

Comment:
In the next refactoring, let's rename anything with visibility to something more specific, i.e. show, or hide in the name.